src/main/java/com/amazonaws/encryptionsdk/keyrings/StandardKeyrings.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except
+ * in compliance with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.amazonaws.encryptionsdk.keyrings;
+
+import javax.crypto.SecretKey;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+
+/**
+ * Factory methods for instantiating the standard {@code Keyring}s provided by the AWS Encryption SDK.
+ */
+public class StandardKeyrings {
+
+    private StandardKeyrings() {
+    }
+
+    /**
+     * Constructs a {@code Keyring} which does local AES-GCM encryption
+     * decryption of data keys using the provided wrapping key.
+     *
+     * @param keyNamespace A UTF-8 encoded value that, together with the key name, identifies the wrapping key.
+     * @param keyName      A UTF-8 encoded value that, together with the key namespace, identifies the wrapping key.
```

Comment:
Nit-pick: these are Java Strings, which means they are internally represented in UTF-16. And that's not really relevant anyway because the underlying `JceKeyCipher` class will call `keyName.getBytes(KEY_NAME_ENCODING)` (where `KEY_NAME_ENCODING == "UTF-8"`) and pass on the `keyNamespace` as a `String`. "A UTF-8 encoded value" really only makes sense if you're describing a `byte[]`.

I do realize the `JceKeyCipher` class has the same misleading comments, so perhaps you can punt fixing this for a separate PR for consistency.

Reply:
Yeah I took that from the RawAesKeyring spec. Should it change there too? 

I'll fix it here for now

Reply:
per updating the spec: I think that an argument can be made that using Java strings and then erroring if there is an issue converting the string to a UTF-8 encoded byte[] later is still following the behavior defined by the specification.

However, there is the issue that in the Java implementation it is possible to initialize keyrings which are unable to produce EDKs (at least for our "standard" keyrings). Is it considered being out of line with the spec to call something a "keyring" if it is unable to perform the behaviors of a keyring? Or is the fact that such a keyring only results in errors mean it's still "in line" with the spec (following the philosophy that: if you can't follow the spec behavior, you MUST fail). For now it is my opinion that we go with the latter approach. Thoughts?

Reply:
Interesting point - the specification should probably say "Unicode string" instead of "UTF-8 encoded value". It would be silly to use byte arrays everywhere instead of the language-native String type.